coralproject/cay
# Modal restyling

## What does this PR do?
- Fixes: https://github.com/coralproject/cay/issues/706
## How do I test this PR?
- Open a saved form with submissions
- Check the design for modals in:
  - Edit Form -> Publish Options
  - Gallery -> Publish Options
  - Gallery -> Edit Submissions

@coralproject/frontend
